plugins/modules/cpanm.py

```diff
@@ -167,6 +174,7 @@ class CPANMinus(ModuleHelper):
             mirror=dict(type='str'),
             mirror_only=dict(type='bool', default=False),
             installdeps=dict(type='bool', default=False),
+            withrecommands=dict(type='bool', default=False),
```

Comment:
```suggestion
            withrecommends=dict(type='bool', default=False),
```

Reply:
So, all good concerning ~~recommands~~ -> recommends, even in filename for the changelog fragments